ruleset.xml

```diff
@@ -1,23 +1,5 @@
 <?xml version="1.0"?>
 <ruleset name="Zend Framework Coding Standard">
     <description>Zend Framework Coding Standard</description>
-
-    <!-- display progress -->
-    <arg value="p"/>
-    <arg name="colors"/>
-
-    <!-- inherit rules from: -->
-    <rule ref="PSR2"/>
-    <rule ref="Generic.Arrays.DisallowLongArraySyntax"/>
-    <rule ref="Generic.Formatting.SpaceAfterNot"/>
-    <rule ref="Squiz.WhiteSpace.OperatorSpacing">
-        <properties>
-            <property name="ignoreNewlines" value="true"/>
-        </properties>
-    </rule>
-    <rule ref="Squiz.WhiteSpace.SuperfluousWhitespace">
-        <properties>
-            <property name="ignoreBlankLines" value="false"/>
-        </properties>
-    </rule>
+    <rule ref="vendor/zendframework/zend-coding-standard/ZendCodingStandard/ruleset.xml"/>
```

Comment:
@weierophinney There was actually an error in that file. the correct location would be `<rule ref="vendor/zendframework/zend-coding-standard/src/ZendCodingStandard/ruleset.xml"/>`.